Cargo.toml

```diff
@@ -8,6 +8,7 @@ members = [
 
     # I/O Crates
     "afpacket",
+    "route-rs-afpacket",
```

Comment:
Nit: I'd like these links to be moved into the runtime, since all the primitive links reside there, and this is low level enough (Read: actually implements Future/Stream) that I think it has a good home there. Also I would like the runtime crate itself to ship with a complete set of tools, which having I/O links is a requirement of.

I think you should put this whole thing in `route-rs-runtime/links/primitive` and have a cfg-linux flag on the relevant mods.

I am leaving it up to you though, I'll accept wherever you want to stash it in the end, hence Nit.